### DISCUSSION TITLE
no boot Amlogic s905w2/s4 tv box transpeed W2

### DISCUSSION THREAD

**Discussion by gadak2022** [devmfc/debian-on-amlogic#37] [Q&A]
Hello, please advise, when booting armbian gives this error, tried all possible dtb files. It gives the same error and won't load
![IMG_20240225_003747](https://github.com/devmfc/debian-on-amlogic/assets/161201523/2ff40321-501a-438b-86be-5583b0df1639)

**Comment by devmfc** [comment]
You have an S905W2 box, but you're trying an s905x4_generic configuration. You really should try one of the S905W2 boxes like tanixw2 or h96maxw2 or s905w2_generic. Maybe you won't have HDMI output, but you should be able to access the box via ssh.

**Comment by gadak2022** [comment]
Alas, I tried all available dtb files from Amlogic folder. All dtb files show ripples of different colors on the monitor. Only this dtb shows the boot progress
![IMG_20240224_093312](https://github.com/devmfc/debian-on-amlogic/assets/161201523/c2bbf45b-3cee-4cff-bc54-340061579a11)

**Reply by devmfc** [reply]
So now you can see clearly on screen that your kernel crashes because you used the wrong config....

That garbage on your screen does not mean your box did not boot properly, only that a framebuffer address is not configured properly for your box.

You should try one of the s905W2 configs and try to ssh to the box or attach a serial adapter to the board to see what is going on. Otherwise I am not able to help you.
